knative-in-mesh/config/knative-eventing/200-allow-api-server-access-webhook.yaml

```diff
@@ -25,3 +25,15 @@ spec:
       app: kafka-webhook-eventing
   ingress:
     - {}
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: kafka-broker-receiver
+  namespace: knative-eventing
+  labels:
+    app: kafka-broker-receiver
+spec:
+  podSelector:
+    matchLabels:
+      app: kafka-broker-receiver
```

Comment:
This is not needed for this example, because the sandbox "tenant" namespace is part of the knative-eventing mesh

Reply:
I know, but I see there potential area of confusion.

Adding `sandbox` to the mesh is not alone, this `NetworkPolicy` is needed too.

I am fine removing this, and adding a bit of clarification on a separate PR

Reply:
@pierDipi updated

Reply:
I'm trying to come up with an example that doesn't give up security by opening up the receiver to the world since it's not really what people want.